[2022-11-12 22:44:03,090] logger_utils.py->print_log line:19 [INFO]包含断言
[2022-11-12 22:44:03,091] logger_utils.py->print_log line:19 [INFO]状态码断言
[2022-11-12 22:44:03,091] logger_utils.py->print_log line:19 [INFO]获取鉴权码-成功业务断言成功
[2022-11-12 22:44:03,147] logger_utils.py->print_log line:19 [INFO]正则表达式"access_token":"(.*?)"可能有误，或者(反例)接口请求失败，未提取到中间变量
[2022-11-12 22:44:03,147] logger_utils.py->print_log line:19 [INFO]jsonpath表达式$..expires_in可能有误，或者(反例)接口请求失败，未提取到中间变量
[2022-11-12 22:44:03,148] logger_utils.py->print_log line:19 [INFO]包含断言
[2022-11-12 22:44:03,148] logger_utils.py->print_log line:19 [INFO]状态码断言
[2022-11-12 22:44:03,149] logger_utils.py->print_log line:19 [INFO]获取鉴权码-appid为空业务断言成功
[2022-11-12 22:44:03,252] logger_utils.py->print_log line:19 [INFO]正则表达式"access_token":"(.*?)"可能有误，或者(反例)接口请求失败，未提取到中间变量
[2022-11-12 22:44:03,252] logger_utils.py->print_log line:19 [INFO]jsonpath表达式$..expires_in可能有误，或者(反例)接口请求失败，未提取到中间变量
[2022-11-12 22:44:03,252] logger_utils.py->print_log line:19 [INFO]包含断言
[2022-11-12 22:44:03,253] logger_utils.py->print_log line:19 [INFO]状态码断言
[2022-11-12 22:44:03,253] logger_utils.py->print_log line:19 [INFO]获取鉴权码-secret为空业务断言成功
